cloud/tasks/tasks_tests/tasks_test.go

```diff
@@ -689,14 +689,19 @@ func TestTasksRunningLimit(t *testing.T) {
 
 			runningLongTaskCount := 0
 			for _, task := range runningTasks {
-				taskState, err := s.storage.GetTask(ctx, task)
+				taskState, err := s.storage.GetTask(ctx, task.ID)
 				require.NoError(t, err)
 
 				if taskState.TaskType == "long" {
 					runningLongTaskCount++
 				}
 			}
-			require.LessOrEqual(t, runningLongTaskCount, inflightLongTaskPerNodeLimit)
+
```

Comment:
Do not introduce unrelated changes in the same commit. Commit shall be an atomic change which transitions from working project state to working project state.